schema/spacewalk/common/tables/namespace.sql

```diff
@@ -20,3 +20,7 @@ COMMENT ON TABLE access.namespace IS 'Namespace definitions to provide access to
 
 CREATE UNIQUE INDEX namespace_ns_mode_uq
 ON access.namespace(namespace, access_mode);
+
+CREATE INDEX namespace_trgm_idx
+ON access.namespace
+USING gist (lower(namespace||description) gist_trgm_ops);
```

Comment:
Maybe we should add a "." after the namespace. Currently, "systemget_details" works, but "system.get_details" doesn't.